flask/app.py

```diff
@@ -919,6 +919,35 @@ def save_session(self, session, response):
         """
         return self.session_interface.save_session(self, session, response)
 
+    def regenerate_session(self, session):
+        """Regenerate the session if it is session id based.  For the default
+        implementation, this is a noop.  Instead of overriding this
+        method we recommend replacing the :class:`session_interface`.
+
+        This should be used upon login to prevent session fixation.
+
+        https://www.owasp.org/index.php/Session_fixation
+
+        :param session: the session to be saved (a
+                        :class:`~werkzeug.contrib.securecookie.SecureCookie`
```

Comment:
We used the same description as `save_session`. I think we should keep it as `SecureCookie` for consistency and update those in another PR